app/Actions/Diagnostics/Pipes/Checks/PHPVersionCheck.php

```diff
@@ -17,26 +21,15 @@ public function handle(array &$data, \Closure $next): array
 
 	private function checkPhpVersion(array &$data): void
 	{
-		// As we cannot test this as those are just raising warnings which we cannot check via Travis.
+		// As we cannot test this as those are just raising warnings which we cannot check via CICD.
 		// I hereby solemnly declare this code as covered !
 		// @codeCoverageIgnoreStart
-
-		// 30 Nov 2019	 => 7.2 = DEPRECATED = ERROR
-		// 28 Nov 2019	 => 7.4 = RELEASED   => 7.3 = WARNING
-		// 26 Nov 2020	 => 8.0 = RELEASED   => 7.4 = WARNING
-		// 6 Dec 2020	 => 7.3 = DEPRECATED = ERROR
-		// 25 Nov 2021	 => 8.1 = Released   => 8.0 = WARNING & 7.4 = ERROR
-		// ! 08 Dec 2022	 => 8.0 = DEPRECATED = ERROR
```

Comment:
You've removed this support table - is it worth linking to https://www.php.net/supported-versions.php when declaring the constants?

Reply:
Fixed with [30fe172](https://github.com/LycheeOrg/Lychee/pull/1671/commits/30fe1722b8f38eb471f8e6bd1bb4b0e4e5ba589a).